### PR TITLE
added new command for updating an env

### PR DESCRIPTION
**Pull request by rahulrayal**
# Odin version(x.y.z)
1.2.0-beta.2

## Features
added update env command for updating an env.
https://dream11.atlassian.net/browse/DEVX-2687

### CODE REVIEW

File: api/environment/env.go [hunk 1]
```diff
@@ -59,6 +59,10 @@ type EnvDeleteResponse struct {
 	EnvResponse EnvDelete `yaml:"resp,omitempty" json:"resp,omitempty"`
 }
 
+type EnvUpdateResponse struct {
```
Comment (sarveshdream11) [inline api/environment/env.go:62]: You are not using this anywhere in the code

File: internal/command/commands/env.go [hunk 2]
```diff
@@ -291,6 +292,51 @@ func (e *Env) Run(args []string) int {
 		return 1
 	}
 
+	if e.Update {
```
Comment (sarveshdream11) [inline internal/command/commands/env.go:295]: Add command helper for update check code line 365

File: internal/command/commands/env.go [hunk 3]
```diff
@@ -291,6 +292,51 @@ func (e *Env) Run(args []string) int {
...
+
+		emptyParameters := emptyParameters(map[string]string{"--name": *name, "--file": *filePath})
+
+		if len(emptyParameters) == 0 {
+			err, parsedConfig := parseFile(*filePath)
+			if err != nil {
+				e.Logger.Error("Error while parsing service file, err: \n" + err.Error())
```
Comment (sarveshdream11) [inline internal/command/commands/env.go:302]: You are not actually parsing service file

File: internal/command/commands/env.go [hunk 4]
```diff
@@ -291,6 +292,51 @@ func (e *Env) Run(args []string) int {
...
+
+			e.Logger.Output(fmt.Sprintf("The new autoDeletionTime is [%s]", envResp.DeletionTime))
+
+			// details, err := yaml.Marshal(envResp)
+
+			// if err != nil {
+			// 	e.Logger.Error(err.Error())
+			// 	return 1
+			// }
+			// e.Logger.Output(fmt.Sprintf("\n%s", details))
```
Comment (sarveshdream11) [inline internal/command/commands/env.go:330]: remove commented code

File: internal/command/commands/env.go (hunk 3 above)
Comment (rahulrayal) [reply internal/command/commands/env.go:302]: we are not taking service file from the user to update an env. we are just parsing the file which has the details of the env to be updated

File: internal/command/commands/env.go (hunk 2 above)
Comment (rahulrayal) [reply internal/command/commands/env.go:295]: I have already added it

File: api/environment/env.go (hunk 1 above)
Comment (rahulrayal) [reply api/environment/env.go]: I have removed it

File: internal/command/commands/env.go (hunk 4 above)
Comment (rahulrayal) [reply internal/command/commands/env.go]: I have removed them

File: internal/command/commands/service.go [hunk 5]
```diff
@@ -363,6 +363,7 @@ func (s *Service) deployReleasedService(envName *string, serviceName *string, se
 
 /*
 validateDeployService
+
```
Comment (dhavalmehta07) [inline internal/command/commands/service.go:366]: remove file from commit

**Review by dhavalmehta07 — APPROVED** [review]

**Review by karan-d11 — APPROVED** [review]

**Pull request merged**